templates/template/README.md

```diff
@@ -122,53 +99,101 @@ Once your directory is clean, you are ready to submit a PR.
 
 #### Versioning Packages
 
-If this `major.minor.patch` (e.g. `0.0.1`) version of the Chart has never been released, reset the `packageVersion` to `01` and the `releaseCandidateVersion` to `00`.
-
-If this `major.minor.patch` (e.g. `0.0.1`) version of the Chart has been released before:
-- If this is the first time you are making a change to this chart for a specific Rancher release (i.e. the current `packageVersion` has already been released in the Live Branch), increment the `packageVersion` by 1 and reset the `releaseCandidateVersion` to `00`.
-- Otherwise, only increment the `releaseCandidateVersion` by 1.
-
-{{ end -}}
-
-{{- if (eq .Template "staging") }}
-
-### Cutting a Release
-
-In the Staging branch, cutting a release involves moving the contents of the `assets/` directory into `released/assets/`, deleting the contents of the `charts/` directory, and updating the `index.yaml` to point to the new location for those assets.
-
-This process is entirely automated via the `make release` command.
+If this `major.minor.patch` (e.g. `0.0.1`) version of the Chart has never been released, reset the `packageVersion` to `01`.
+
+If this `major.minor.patch` (e.g. `0.0.1`) version of the Chart has been released before, increment the `packageVersion`.
+
+### Porting over Charts / Assets from another Branch
+
+In the Staging branch, porting over charts from another branch (e.g. `dev-v2.x+1`) requires you to copy the contents of that branch into your Staging branch, which can be done with the following simple Bash script. However, you will need to manually regenerate the Helm index since you only want the index.yaml on the Staging branch to be updated to include the new chart.
```

Comment:
I'm a little confused on this part. Aren't dev and staging the same now ? Meaning isn't dev-v2.5 the staging branch ? The goal of this script I'm guessing is to move across release lines, not entirely clear. 
As for the whole bash script below, can't you just copy paste a folder from one versions branch to another ? Script below is not intuitive, I don't get the point of the forked branch / port.

Reply:
Yeah dev and staging are the same now. The goal of this script is, let's say, moving from `dev-v2.6` to `dev-v2.5`.

You don't want to necessarily maintain two versions of the chart in `packages/`, so you can make the commit to dev-v2.6 with the `packages/` changes and simply port the charts, assets, and index.yaml over to `dev-v2.5`.

Or, if we use @nickgerace 's idea of having a single dev-v2.x branch that contains packages, this would be the normal process to make that version available in `dev-v2.5` or `dev-v2.6` branches (e.g. identical to what we do today with making a commit to `master` and then porting it to `release/v2.5` and `release/v2.6`.

Copying and pasting a folder isn't sufficient since you also need to regenerate the Helm index. But basic idea is the same yeah; you need to copy the charts/, the assets/ and regenerate the Helm index to port over just the generated resources.

You can't do a simple cherry-pick across branches since the index.yaml may have conflicts.

Reply:
No cherry-pick makes sense, as does index.yaml regeneration. But why not just copy `package/logging` folder, swap branches, and then run `make charts` and `make validate` again ?

Reply:
You’ll be committing 2 versions of the logging chart. This is fine, as long as you keep both copies up to date, but if all you want is to make the asset available it makes more sense to have a single source of truth.